src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(Colors.valueOf(color));
+        this.radius = Math.round(Math.abs(radius));
```

Comment:
Rounding the radius using `Math.round()` might not be necessary unless specified by the task requirements. Ensure this aligns with the intended functionality.